_posts/2019-07-19-p2p-data-sync-for-mobile.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-name:  "P2P Data Sync for Mobile"
-title:  "P2P Data Sync for Mobile"
-date:   2019-07-19 12:00:00 +0800
+name: 'P2P Data Sync for Mobile'
+title: 'P2P Data Sync for Mobile'
+date: 2019-07-19 12:00:00 +0800
 author: oskarth
 published: true
 permalink: /p2p-data-sync-for-mobile
-categories: research
+category: research
```

Comment:
Can we keep this to be categories so it allows for multiple categories?

Reply:
Sure, changed.